container/acl/acl_basic.go

```diff
@@ -0,0 +1,278 @@
+package acl
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+// Basic represents basic part of the NeoFS container's ACL. It includes
+// common (pretty simple) access rules for operations inside the container.
+// See NeoFS Specification for details.
+//
+// One can find some similarities with the traditional Unix permission, such as
+//  division into scopes: user, group, others
+//  op-permissions: read, write, etc.
+//  sticky bit
+// However, these similarities should only be used for better understanding,
+// in general these mechanisms are different.
+//
+// Instances can be created using built-in var declaration, but look carefully
+// at the default values, and how individual permissions are regulated.
+// Some frequently used values are presented in exported variables.
+//
+// Basic instances are comparable: values can be compared directly using
+// == operator.
+type Basic struct {
+	bits uint32
+}
+
+// FromBits decodes Basic from the numerical representation.
+//
+// See also Bits.
+func (x *Basic) FromBits(bits uint32) {
+	x.bits = bits
+}
+
+// Bits returns numerical encoding of Basic.
+//
+// See also FromBits.
+func (x Basic) Bits() uint32 {
```

Comment:
if we still have that getter, wouldnt it be a good idea to return the table that you have deleted? mb in the `doc` file

Reply:
I don't want to duplicate NeoFS Spec since it becomes hard to support it. Also `Bits` method is utility for client and server side conversions and no so usable for other cases.